iRacing/iRacing.cs

```diff
@@ -1915,14 +1858,16 @@ public void DataUpdate()
                 double weight = 1600 / Math.Log(2);
                 double posCorr = (classOpponents / 2 - realPosition) / 100;
 
-                GameReaderCommon.Opponent opponent = Base.gameData.NewData.Opponents.FirstOrDefault(x => x.CarClass == myClass);
-                if (opponent != null)
-                {
-                    iratings.Add(opponent.IRacing_IRating);
-                }
-                else
+                for (int i = 0; i < opponents; i++)
                 {
-                    iratings.Add(0);
+                    if (Base.gameData.NewData.Opponents[i].CarClass == myClass)
+                    {
+                        iratings.Add(Base.gameData.NewData.Opponents[i].IRacing_IRating);
+                    }
+                    else
+                    {
+                        iratings.Add(0);
+                    }
```

Comment:
So this looks like it fixed the iRating issue, hadn't noticed it was broken. you know when this happened? 😅

Reply:
this must have happend on Nov, 10th when I made the refactoring. I messed up the loop logic